### PR TITLE
Button text component

### LINKED ISSUE #39: Primary and Secondary `ButtonText` Components
image
Make these two separate `styled-components` buttons

### PR DESCRIPTION
**Pull request by meleongg**
Closes #39 hopefully, not sure if I did what was expected

### CODE REVIEW

**Comment by samanthatuhseng** [comment]
I feel like the hovers should be states instead of like a toggle?

**Comment by meleongg** [comment]
Couple of Questions:

1. Should I create a new "Small Button" template in Button.stories.jsx with the associated JS (in Button.js) and CSS (in ButtonCSS.js) since these 2 text components are the same size as the Small Buttons in the Figma file. Currently using the LargeButton template. 

2. I made the font size of my buttons to be 1.1250rem, the equivalent of 18 px (as indicated on Figma file), which is currently larger than the Large button, so not sure if that's what we wanted.

**Comment by naijwu** [comment]
1. It would make sense to but @ajo01 what do you think of this?
2. I think the font size should be 1.125 rem / 18 px, will need to change the LargeButton font size

**Comment by ajo01** [comment]
1. Creating another template would create another component folder in storybook. Ideally we'd keep the current template but rename it to Button and make its size controllable by its props instead (the logic for this currently doesn't exist though). Currently the Button component is a LargeButton component that is actually using some of SmallButton styles. For this PR we should aim to refactor it so the Button component is imported/exported as a Button component, and we can optionally fix the LargeButton/SmallButton styles in another PR. This way the Header component will compile too.
I'll send you a diff patch for this. 
2. We should follow the figma files when possible. You'd be right to use 1.125 rem for the font size. Just make sure to update the font sizes for all the other button styles. 
@phoenixthefirebird FYI

File: src/components/button/Button.js [hunk 1]
```diff
@@ -1 +1,87 @@
...
+  SecondaryCSS,
+  PrimaryTextCSS,
+  SecondaryTextCSS,
+} from "./ButtonCSS";
+
+// Button
+const Button = (props) => {
+  const getVariantStyles = (p) => {
+    const inactiveText = `color: ${colors.grey600}`;
+
```
Comment (ajo01) [inline src/components/button/Button.js:18]: Add `cursor: default` to inactiveText

**Review by ajo01 — CHANGES_REQUESTED** [review]
Looking at the other button components (button nav & pill button), it seems like the people responsible for them will make them separate UI components. Following that, we should make changes so the button text component is its own component with its own ButtonText.js and stories file. Look at Derek's PR for reference https://github.com/nwplus/Aurora/pull/49

**Comment by meleongg** [comment]
Screen Shot 2022-07-13 at 11 16 55 AM
Screen Shot 2022-07-13 at 11 17 07 AM

NOTE: made background darker via chrome devtools to actually see the button

**Review by ajo01 — APPROVED** [review]
LGTM!

**Pull request merged**